cli/test/smokehouse/test-definitions/dobetterweb.js

```diff
@@ -242,6 +242,25 @@ const expectations = {
         },
       ],
     },
+    ImageElements: {
+      _includes: [{
+        src: 'http://localhost:10200/dobetterweb/lighthouse-1024x680.jpg?iar2',
+        srcset: '',
+        displayedWidth: 120,
+        displayedHeight: 80,
+        attributeWidth: '120',
+        attributeHeight: '80',
+        naturalDimensions: {
+          width: 1024,
+          height: 678,
```

Comment:
notice the image is called `lighthouse-1024x680.jpg`... :P

Reply:
So the file name is just misleading?

Reply:
> So the file name is just misleading?

yes, but to be fair, it's copied from the byte-efficiency smoke test [image of the same name](https://github.com/GoogleChrome/lighthouse/blob/a7891f3637c74317316dfd17faab2579a1017f38/cli/test/fixtures/byte-efficiency/lighthouse-1024x680.jpg), which is (apparently) also actually 1024x678, added wayyyy back in #1635

Reply:
Yeah not suggesting we change it here, just wanted to confirm